hack/build.sh

```diff
@@ -5,7 +5,9 @@ SUPERNODE_BINARY_NAME=supernode
 PKG=github.com/dragonflyoss/Dragonfly
 BUILD_IMAGE=golang:1.12.6
 VERSION=$(git describe --tags | cut -c 2-)
```

Comment:
>remove the character $ and string -14-g61aa7a9.
Usually a release is built on a tag, and the version value is the tag name without its prefix v.